### ISSUE TITLE
Basic Connection with clickhouse

### ISSUE THREAD

**Issue by happyharry123** [apla/node-clickhouse#2]
Hey apla,
I have a  trouble on connecting clickhouse with nodejs. I am now working with you sample code and some questions pop-up with the clickhouse options. I just want to know it is correct of the options format? 
Thank you a lot for this modules with clickhouse! 

`var ClickHouse = require ('@apla/clickhouse');
var options = {
  host: '160.xx.xx.xxx:8123',
  auth:{
    user:'abc123',
    password:'paswword123'
  },
};
var ch = new ClickHouse (options);
var stream = ch.query ("SELECT * from xxx limit 1");

// or collect records yourself
var rows = [];

stream.on ('metadata', function (columns) {
  // do something with column list
});
`
I got an error about 

> first argument must be a string buffer arraybuffer array or array-like object

**Comment by vickas777** [comment]
Think your auth is not correct. This config will help.
```
var options = {
  host: '160.xx.xx.xxx',
  port: 8123
  auth: 'abc123:paswword123',
  dataObjects:  'FORMAT JSON'
};
```

**Comment by happyharry123** [comment]
Thank you Vickas.
Your solution is correct, you save my day! Thank you.

**Comment by sultra** [comment]
Hey,
Can you give an example about "statement" structure when insert row data?

**Comment by nezed** [comment]
Solved, probably must be fixed with #34
